app/api/tickets/rate/route.ts

```diff
@@ -14,12 +15,19 @@ export async function POST(req: Request) {
   const { ticketId, rate } = (await req.json()) as TAPITicketsRate
   const { error } = await supabaseAdmin.from("tickets").update({ is_open: false, rate: rate }).eq("id", ticketId)
   if (error) return NextResponse.json({ error: `Error in api/tickets/route.ts\n ${error.message}` }, { status: 400 })
+  // to don't show ticket it on support side
   await pusherServer.trigger("tickets", "tickets:close", {
     id: ticketId,
```

Comment:
Here is may be unnecessary to pass data because you never use this data - you just `setMessages([])`

Reply:
It necessary because without it I got ticketId undefined regardless for pusher I have the same event but in another channel - so I need to rename event even if I trigger pusher for another channel

Reply:
About data you'r right - it unnecessary